jkube-kit/doc/src/main/asciidoc/inc/enricher/triggers-annotation/_jkube_triggers_annotation.adoc

```diff
@@ -0,0 +1,16 @@
+
+[[jkube-triggers-annotation]]
+==== jkube-triggers-annotation
+
+OpenShift resources like BuildConfig and DeploymentConfig can be automatically triggered by changes to ImageStreamTags. However, plain Kubernetes resources don't have a way to support this kind of triggering. You can use `image.openshift.io/triggers` annotation in OpenShift to request triggering. Read OpenShift docs for more details : https://access.redhat.com/documentation/en-us/openshift_container_platform/4.5/html/images/triggering-updates-on-imagestream-changes[Triggering updates on ImageStream changes]
```

Comment:
nit: Maybe use the latest released OpenShift documentation link https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/images/triggering-updates-on-imagestream-changes